src/model/missions.nit

```diff
@@ -37,10 +37,14 @@ class Mission
 	var parents = new Array[String]
 	var stars = new Array[MissionStar]
 
+	# Number of points to solve the mission (excluding stars)
+	var solve_reward: Int = 1 is writable
```

Comment:
`reward` already exist and means *total reward of the mission incl stars*. Moreover it is already used by angular, so renaming will be more complex.